src/main/resources/hudson/plugins/copyartifact/monitor/LegacyJobConfigMigrationMonitor/resources.css

```diff
@@ -27,7 +27,8 @@
 .legacy-copy-artifact table tr td.first-column {
     padding: 0;
 }
-.legacy-copy-artifact table tr td img.first-column-icon {
+.legacy-copy-artifact table tr td.first-column .icon-warning {
+    box-sizing: content-box;
```

Comment:
* layout:icon doesn't look accept extra class. I'll work for that in https://issues.jenkins.io/browse/JENKINS-68286
* Jenkins set `box-sizing` to `border-box`: https://github.com/jenkinsci/jenkins/blob/jenkins-2.343/war/src/main/webapp/bootstrap/css/bootstrap.css#L200
    * And specifying width 16px and padding 4px results icon width 12px: https://developer.mozilla.org/ja/docs/Web/CSS/box-sizing
    * So switched to content-box here.

Reply:
Further think, applying `padding` to icon is strange manner. I'll try `margin` instead.